web/packages/teleport/src/Bots/List/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useState, useEffect } from 'react';
+import styled, { useTheme } from 'styled-components';
+import { Link } from 'react-router-dom';
+import { GitHubIcon, SpaceliftIcon, AnsibleIcon } from 'design/SVGIcon';
+import { ResourceIcon } from 'design/ResourceIcon';
+import {
+  Description,
+  DetailsTab,
+  Feature,
+  FeatureContainer,
+  FeatureProps,
+  FeatureSlider,
+  Title,
+} from 'shared/components/EmptyState/EmptyState';
+import { Box, ButtonPrimary, Flex, H1, Image, Text } from 'design';
+
+import cfg from 'teleport/config';
+import { DisplayTile } from 'teleport/Bots/Add/AddBotsPicker';
+
+import elimiateSecretsImage from './eliminate-secrets.svg';
+import elimiateSecretsLightImage from './eliminate-secrets-light.svg';
+
+import controlWorkflowsImage from './control-workflows.svg';
+import controlWorkflowsLightImage from './control-workflows-light.svg';
+
+import argoCD from './argocd.png';
+
+const maxWidth = '1204px';
+
+export function EmptyState() {
+  const [currIndex, setCurrIndex] = useState(0);
+  const [intervalId, setIntervalId] = useState<any>();
+
+  function handleOnClick(clickedIndex: number) {
+    clearInterval(intervalId);
+    setCurrIndex(clickedIndex);
+    setIntervalId(null);
+  }
+
+  useEffect(() => {
+    const id = setInterval(() => {
+      setCurrIndex(latestIndex => (latestIndex + 1) % 3);
+    }, 3000);
+    setIntervalId(id);
+    return () => clearInterval(id);
+  }, []);
+
+  return (
+    <Box mt={4} data-testid="bots-empty-state">
+      <Box mb={3}>
+        <H1 mb={3}>What are Bots?</H1>
+        <Text css={{ maxWidth }}>
+          Static keys and API keys in your automated workflows are the target of
+          hackers and are one of the key sources of security breaches.
```

Comment:
Slight change to the copy here so we don't say "key" three times in one sentence might be nice 😓 

```suggestion
          hackers and are one of the primary sources of security breaches.
```